src/stories/Library/promo-bar/promo-bar.scss

```diff
@@ -0,0 +1,16 @@
+.promo-bar {
+  min-height: $s-2xl;
+  background-color: $c-global-secondary;
+  display: flex;
+  align-items: center;
+  gap: $s-md;
+
+  &.promo-bar--stick-list-item {
```

Comment:
maybe **.promo-bar--sticky** is enough here? Just a note if you wanna shorten the naming 😉